README.md

```diff
@@ -101,43 +101,61 @@ the `WORKSPACE` file.
 ### Deployments
 
 Buildfarm can be used as an external repository for composition into a deployment of your choice.
-
-Add the following to your WORKSPACE to get access to buildfarm targets, filling in the commit and sha256 values:
+See also the documentation site in the [Worker Execution Environment](https://bazelbuild.github.io/bazel-buildfarm/docs/architecture/worker-execution-environment/) section.
+ 
+Add the following to your `MODULE.bazel` to get access to buildfarm targets, filling in the `<COMMIT-SHA>` values:
 
 ```starlark
-load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
-
-BUILDFARM_EXTERNAL_COMMIT = "<revision commit id>"
-BUILDFARM_EXTERNAL_SHA256 = "<sha256 digest of url below>"
-
-http_archive(
-    name = "build_buildfarm",
-    strip_prefix = "bazel-buildfarm-%s" % BUILDFARM_EXTERNAL_COMMIT,
-    sha256 = BUILDFARM_EXTERNAL_SHA256,
-    url = "https://github.com/bazelbuild/bazel-buildfarm/archive/%s.zip" % BUILDFARM_EXTERNAL_COMMIT,
+bazel_dep(name = "build_buildfarm")
+git_override(
+    module_name = "build_buildfarm",
+    commit = "<COMMIT-SHA>",
```

Comment:
going to land this, but I want to at some point do the 'generate these scripts as a result of the latest release', ideally for 2.11

Reply:
or eventually we put buildfarm in the BCR.. https://registry.bazel.build/